src/pages-and-resources/discussions/DiscussionsSettings.jsx

```diff
@@ -52,7 +52,7 @@ function DiscussionsSettings({ courseId, intl }) {
     } else {
       history.push(discussionsPath);
     }
-  }, [discussionsPath]);
+  }, [isFirstStep]);
```

Comment:
why `discussionsPath` is not in the deps?

Reply:
Oh, derp.  You're right - I've screwed it up further. :)

Reply:
Fixed, good catch!